cocos/core/utils/jsb-utils.ts

```diff
@@ -146,8 +139,8 @@ export function updateChildrenForDeserialize (node: Node) {
         return;
     }
 
-    // @ts-expect-error: jsb related codes.
-    node._setChildren(children);
+    // TODO: `_setChildren` is only implemented on native platforms. @dumganhar
+    (node as any)._setChildren(children);
```

Comment:
Yes, it's a private method only for native. It's used to synchronize JS `children` array to c++.

Reply:
got it, I've removed this TODO item, and mark it as `NOTE`
https://github.com/cocos/cocos-engine/pull/14572/commits/551eb3a839ae0ae75280b7b8828bf67df9f934ec